tests/Makefile

```diff
@@ -68,6 +72,13 @@ idempotent_seq: $(BIN)
 
 idempotent: idempotent_par
 
+transactions: $(BIN)
+	for _test in 0098 0101; do TESTS=$$_test ./run-test.sh ./$(BIN) ; done
```

Comment:
The run-test syntax changed, and the list of tests is incomplete. This is a rather pointless target.